.github/workflows/tests.yml

```
@@ -0,0 +1,34 @@
+name: Tests
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)